### PR TITLE
Move files in assets to samples

### PR DESCRIPTION
**Pull request by feihugis**

### CODE REVIEW

File: demo.ipynb [hunk 1]
```diff
@@ -241,6 +241,15 @@
...
    "version": "3.6.5"
+  },
+  "pycharm": {
+   "stem_cell": {
+    "cell_type": "raw",
+    "metadata": {
+     "collapsed": false
+    },
+    "source": []
+   }
```
Comment (xuhdev) [inline demo.ipynb]: You sure  wanna increase these execution_count by 1 and add this pycharm metadata?

File: demo_gdpr.ipynb [hunk 2]
```diff
@@ -315,6 +315,15 @@
...
    "version": "3.6.5"
+  },
+  "pycharm": {
+   "stem_cell": {
+    "cell_type": "raw",
+    "metadata": {
+     "collapsed": false
+    },
+    "source": []
+   }
```
Comment (xuhdev) [inline demo_gdpr.ipynb]: Same as above.

**Comment by splovyt** [comment]
Your assets are currently in the `max-assets-dev` bucket, but if you think these are final you can move these to `max-assets-prod`. A second minor remark is that according to the skeleton guidelines we should store the demo notebooks in the `samples` dir. 

LGTM in general, thanks Fei!

File: demo.ipynb (hunk 1 above)
Comment (feihugis) [reply demo.ipynb]: Good catch! Will change them.

**Comment by feihugis** [comment]
@splovyt Thanks for the quick review!

> Your assets are currently in the `max-assets-dev` bucket, but if you think these are final you can move these to `max-assets-prod`. 

Yeah, will move the model to `max-assets-prod` in the next PR.

> A second minor remark is that according to the skeleton guidelines we should store the demo notebooks in the `samples` dir.

Do you mean this repo: https://github.ibm.com/CODAIT/MAX-skeleton/tree/master?

**Comment by splovyt** [comment]
> Do you mean this repo: https://github.ibm.com/CODAIT/MAX-skeleton/tree/master?

That's the old archived one. The public one is up to date:
https://github.com/IBM/MAX-Skeleton

**Comment by feihugis** [comment]
Thanks, @splovyt. Will change it.

**Comment by feihugis** [comment]
@splovyt @xuhdev The comments are addressed here (https://github.com/IBM/MAX-Facial-Recognizer/pull/23/commits/1fe88f227113917b51f585b07953296a8c2d4445). Please have another look!

File: samples/README.md [hunk 3]
```diff
@@ -1,10 +1,4 @@
-# Asset Details
-
-## Model files
-
-Model files are from the [FaceNet model](https://github.com/davidsandberg/facenet) where they are available under an [MIT License](https://github.com/davidsandberg/facenet/blob/master/LICENSE.md).
-
-_Note_ the model files are hosted on [IBM Cloud Object Storage](http://max-assets.s3.us.cloud-object-storage.appdomain.cloud/facial-recognizer/1.0/assets.tar.gz).
```
Comment (bdwyer2) [inline samples/README.md]: Is this information still provided somewhere in this repository?

Comment (splovyt) [reply samples/README.md]: In the main readme

Comment (feihugis) [reply samples/README.md]: Yeah, the main readme has the model license info, so I removed this one.

**Pull request merged**